packages/website/src/components/linter/createLinter.ts

```diff
@@ -160,7 +167,37 @@ export function createLinter(
       console.log('[Editor] Updating', fileName, compilerOptions);
       parser.updateConfig(compilerOptions);
     } catch (e) {
-      console.error(e);
+      if (e instanceof Error) {
+        const error = {
+          group: 'Typescript',
+          items: e.message
+            .trim()
+            .split('\n')
+            .map((message: string) => {
```

Comment:
Do you have an example of a message that needs to be split? I've only been able to make versions with a single reported error at a time.

Reply:
I'm not sure if this is realistic, but I used split because I was able to generate two errors with the settings below.

```
{
  "compilerOptions": {
    "moduleResolution": "classic",
    "resolveJsonModule": true,
    "incremental": true,
    "outFile": "dist/bundle.js"
  }
}
```

<img width="871" alt="스크린샷 2025-04-12 오전 11 12 03" src="https://github.com/user-attachments/assets/8746d27d-db5b-438f-b72f-6e3fa2a21be0" />